recipes/poliastro/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+pip install --no-deps .
```

Comment:
Well, when it comes to Mac and Linux. We will fold this into `script` under `build` in the `meta.yaml` file. Unfortunately, we can't do this for Windows due to a `conda build` bug.